sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Changed `ServiceBusMessageBatch.tryAddMessage()` to be thread-unsafe.([#25910](https://github.com/Azure/azure-sdk-for-java/issues/25910))
```

Comment:
Should we say "Improved performance of `ServiceBusMessageBatch.tryAddMessage()`. The class is not thread-safe."? :-)
You can wait for Anu's comment.

Also have a space after period.

Reply:
So method was never intended to be thread-safe and the old code itself was "broken", so I wonder changelog could just say something like 

**Removed the incorrect use of lock primitives from `ServiceBusMessageBatch.tryAddMessage()` implementation and documented that this API is not threadsafe.**

the linked git-issue has details why the code was broken, and we're correcting it, combining that with the JavaDoc note on thread-unsafe should make it clear. What do you think?

Reply:
Thanks Anu, it sounds more accurate for the scope of changes. I'll update it.